src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -331,22 +325,5 @@ public StackItem ToStackItem(ReferenceCounter referenceCounter)
                 return new Struct(referenceCounter) { Registered, Votes };
             }
         }
-
-        private sealed class GasRecord : List<(uint Index, BigInteger GasPerBlock)>, IInteroperable
```

Comment:
It is removed?

Reply:
It was moved to an iterator, in order to avoid the serialization limits.

Reply:
I can revert this change, and fix only the serialization case, we can find a different solution for store more items than expected.

Reply:
You can create another PR for these changes.

Reply:
Done